enterprise/frontend/src/embedding-sdk/hooks/private/use-sdk-dashboard-params.ts

```diff
@@ -55,17 +54,11 @@ export type SdkDashboardDisplayProps = {
 } & CommonStylingProps;
 
 export const useSdkDashboardParams = ({
-  dashboardId: dashboardIdProp,
   withDownloads,
   withTitle,
   withCardTitle,
   hiddenParameters,
 }: SdkDashboardDisplayProps) => {
-  const { id: dashboardId, isLoading = false } = useValidatedEntityId({
-    type: "dashboard",
-    id: dashboardIdProp,
-  });
```

Comment:
`/dashboard/:id` supports an entity id now, so no need for frontend translation.